### ISSUE TITLE
Fine-Tuning process

### ISSUE THREAD

**Issue by ugmSorcero** [artitw/text2text#9]
Hi!
I would like to know the process of fine-tuning UniLM with inverted SQUAD (hardware, training time, number of steps, parameters, etc.)
Would that be possible?
Thanks in advance!

**Comment by thusithaC** [comment]
Yes, I have the same question. The repo is extremely useful and provides good quality results and easy to use and setup compared to some purely research based github repos. 

However, this might be a naive question, but does this repo even include the code needed to train the .bin file. Would love to recreate this in other languages, so it would be extremely helpful if a re-training guide can be included in the readme, with links to the source datasets.
